go/vt/binlog/tables_filter.go

```diff
@@ -11,7 +11,10 @@ import (
 	"github.com/youtube/vitess/go/vt/binlog/proto"
 )
 
-var STREAM_COMMENT = "/* _stream "
+var (
```

Comment:
Are you planning on moving the other annotations (done by vttablet) into this library as well? Would be cool I think to have them all in one place.